### PR TITLE
`emptyCache` after `FusionCompileIndexType_CUDA`

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

File: test/test_gpu3.cpp [hunk 1]
```diff
@@ -7784,130 +7784,135 @@ TEST_F(NVFuserTest, FusionTypePromotionATenConsistency_CUDA) {
...
 TEST_F(NVFuserTest, FusionCompileIndexType_CUDA) {
-  Fusion fusion;
-  FusionGuard fg(&fusion);
-
-  auto tv0 = makeSymbolicTensor(1, DataType::Bool);
-  fusion.addInput(tv0);
-
-  auto tv2 = neg(tv0);
-  fusion.addOutput(tv2);
+  {
```
Comment (zasdfgbnm) [inline test/test_gpu3.cpp:7787]: No code change, just put it into a scope and reformatting.

**Comment by zasdfgbnm** [comment]
This do help with https://github.com/NVIDIA/Fuser/issues/48 on my system

**Review by naoyam — APPROVED** [review]
Why doesn't the caching allocator automatically clear caches when necessary?

**Comment by zasdfgbnm** [comment]
> Why doesn't the caching allocator automatically clear caches when necessary?

I don't know /shrug

**Pull request merged**